### DISCUSSION TITLE
FastAPI security with third-party libriaries

### DISCUSSION THREAD

**Discussion by tobyweb3x** [fastapi/fastapi#5991] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI
```

### Description

I asked tiangolo a question on Twitter and he replied to bring it here so everyone can contribute

![image](https://user-images.githubusercontent.com/73890077/218273160-3f2791a2-8973-4a0b-9396-9ce7b598376c.png)

### Operating System

Linux

### FastAPI Version

0.9

### Python Version

3.10

**Comment by mr-st0rm** [comment]
what do you mean by security?

FastApi has its own security tools like CARS, BearerAuth, Oauth and so on.
